### PR TITLE
Taras

### PR DESCRIPTION
**Pull request by TarasRudyk**
Please review my changing in code, removed scroll in small container(stage) limited  max and min height. In result the height of container is still  changing if amount of tasks in week bigger then in month but in limitation. In small amount of task the scroll don't appear

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
     "eslint": "^2.13.1",
     "eslint-plugin-react": "^4.3.0",
     "faker": "^3.1.0",
-    "react": "^15.1.0",
+    "js-yaml": "^3.6.1",
```
Comment (epaminond) [inline package.json]: Why did you add it?

File: client/stylesheets/main.css [hunk 2]
```diff
@@ -1,11 +1,13 @@
 /* Stages timeline */
 .stage {
   display: flex;
-  flex: 1 100%;
+  flex:1 100%;
 }
 .stages-list {
   display: flex;
-  flex: 100%;
+  /*flex: 100%;*/
```
Comment (epaminond) [inline client/stylesheets/main.css]: Please, don't leave code commented out (we have git history anyway).

File: package.json (hunk 1 above)
Comment (TarasRudyk) [reply package.json]: Mistake, reverted to previous.

File: client/stylesheets/main.css [hunk 3]
```diff
@@ -1,11 +1,12 @@
 /* Stages timeline */
 .stage {
   display: flex;
-  flex: 1 100%;
+  flex:1 100%;
```
Comment (epaminond) [inline client/stylesheets/main.css]: Why removing space here?

File: client/stylesheets/main.css [hunk 4]
```diff
@@ -51,6 +54,7 @@
   flex: 0%;
 }
 
+
```
Comment (epaminond) [inline client/stylesheets/main.css]: Line seems to be redundant.

File: client/stylesheets/main.css [hunk 5]
```diff
@@ -43,6 +44,8 @@
   width: 100%;
   border-width: 1px;
   white-space: nowrap;
+  /*min-height: 15em;*/
```
Comment (epaminond) [inline client/stylesheets/main.css]: If it's not needed, please remove it.

File: package.json [hunk 6]
```diff
@@ -74,4 +74,4 @@
       "new-cap": 0
     }
   }
-}
+}
```
Comment (epaminond) [inline package.json]: Best practice is to leave one empty line at the end of file.

**Review by epaminond — CHANGES_REQUESTED** [review]
